app/models/concerns/eto/pathways_answers.rb

```diff
@@ -0,0 +1,228 @@
+      end
+
+      def rrh_th_desired_answer
+        # 2/17/2020
+        # 9F
+        return false unless hmis_assessment.pathways?
+
+        relevant_section = section_starts_with('Why do people take RRH')
+        answer_from_section(relevant_section, '9F.')&.downcase == 'yes'
+      end
+
+      def rrh_assessment_contact_info_answer
+        # 2/17/2020
+        nil # FIXME: this does not appear to come through the API at this time.
+      end
+
+      def income_maximization_assistance_requested_answer
+        # 2/17/2020
+        # 10C
+        return false unless hmis_assessment.pathways?
+
+        relevant_section = section_starts_with('Key Points to Share with the Participant Regarding Household History')
+        answer_from_section(relevant_section, '10C.')&.downcase == 'yes'
+      end
+
+      def income_total_annual_answer
+        # 2/17/2020
+        # 6A
+        return false unless hmis_assessment.pathways?
+
+        relevant_section = section_starts_with('Key Points to Share with the Participant Regarding Unit Preferences')
+        answer_from_section(relevant_section, '6A.')&.to_i
+      end
+
+      def pending_subsidized_housing_placement_answer
+        # 2/17/2020
+        # 5D
+        return false unless hmis_assessment.pathways?
+
+        relevant_section = section_starts_with('Section 5 - Household Composition')
+        answer_from_section(relevant_section, '5D.')&.downcase == 'yes'
+      end
+
+      def domestic_violence_answer
+        # 2/17/2020
+        # 2B
+        return false unless hmis_assessment.pathways?
+
+        relevant_section = section_starts_with('Key Points to Share with the Participant regarding Safety & Urgent Health Conditions')
+        answer_from_section(relevant_section, '2b.')&.downcase == 'yes'
+      end
+
+      def interested_in_set_asides_answer
+        # 2/17/2020
+        # 6I
+        return false unless hmis_assessment.pathways?
+
+        relevant_section = section_starts_with('6I.')
+        answer_from_section(relevant_section, 'I confirm my interest in signing up for the homeless set aside units')&.downcase == 'yes'
+      end
+
+      def required_number_of_bedrooms_answer
+        # 2/17/2020
+        # 6C
+        return false unless hmis_assessment.pathways?
+
+        relevant_section = section_starts_with('Key Points to Share with the Participant Regarding Unit Preferences')
+        answer_from_section(relevant_section, '6c.')&.to_i
+      end
+
+      def required_minimum_occupancy_answer
+        # 2/17/2020
+      end
+
+      def requires_wheelchair_accessibility_answer
+        # 2/17/2020
+        # 6D
+        return false unless hmis_assessment.pathways?
+
+        relevant_section = section_starts_with('Key Points to Share with the Participant Regarding Unit Preferences')
+        answer_from_section(relevant_section, '6d.')&.downcase&.include?('wheelchair') || false
+      end
+
+      def requires_elevator_access_answer
```

Comment:
Both this and above are `6D`?

Reply:
Yes, the question on the TouchPoint comes back pipe delimited.